vizro-ai/docs/pages/user-guides/add-generated-chart-usecase.md

```diff
@@ -99,8 +99,8 @@ This object must then be passed to the `figure` argument of the Vizro [Graph](ht
         page = vm.Page(
             title = 'Demographics',
             components = [
-                vm.Graph(id='bubble chart', figure=custom_chart(df)),
```

Comment:
@maxschulz-COL I've rolled back the removal of `id` as you requested in the langchain page of the Vizro-AI docs. Shall I also roll out these 4 removals in this page?

Reply:
Thanks, good to know. In which case, it's all ready and I'll merge.